task/openwhisk/0.1/openwhisk.yaml

```diff
@@ -3,7 +3,22 @@ apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: openwhisk
+  lables:
```

Comment:
typo labels

Reply:
We must work on a tool to validate this, otherwise these typos are quite easy to miss 
cc: @vdemeester @bobcatfish @afrittoli